### DISCUSSION TITLE
How to Restore

### DISCUSSION THREAD

**Discussion by seanbts** [dec0dOS/zero-ui#97] [Q&A]
Hi,
Can you tell me the directory location of the controller and peers in ubuntu, i ran the update command and now when i login to the web interface none of our networks / peers are showing, maybe i over wrote when doing the update command....is there a directory i can check for this data to restore?

Thanks for your help.

**Comment by mfld-pub** [comment]
You can try to reverse engineer the[ backup instructions](https://github.com/dec0dOS/zero-ui#backup) for hints on where the good bits are stored.

**Comment by cschmidtoptima** [comment]
I tried to restore my data on a new server and I can see all the networks in the control panel but when I try to create and connect to a new one it says in the client "NOT_FOUND" and in the controller log: 

> ERROR: unable to configure virtual network port: could not open TUN/TAP device: No such file or directory
